<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>musteribilgileri</class>
 <widget class="QDialog" name="musteribilgileri">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Müşteri Bilgileri</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../images.qrc">
    <normaloff>:/icons/win_icon.png</normaloff>:/icons/win_icon.png</iconset>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>381</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Yeni Müşteri Kaydı İçin Tıklayınız</string>
   </property>
   <property name="icon">
    <iconset resource="../images.qrc">
     <normaloff>:/icons/kayit_icon.png</normaloff>:/icons/kayit_icon.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>381</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Mevcut Müşteri Tablosunu Görmek İçin Tıklayınız</string>
   </property>
   <property name="icon">
    <iconset resource="../images.qrc">
     <normaloff>:/icons/table_icon.png</normaloff>:/icons/table_icon.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>181</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Müşteri Kodu :  </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_3">
        <property name="text">
         <string>ARA</string>
        </property>
        <property name="icon">
         <iconset resource="../images.qrc">
          <normaloff>:/icons/search_icon.png</normaloff>:/icons/search_icon.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Müşteri Adı :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="MusteriAdi">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Müşteri Soyadı :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="MusteriSoyadi">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Müşteri Adresi :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="MusteriAdresi">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../images.qrc"/>
 </resources>
 <connections/>
</ui>
